pkg/util/ecs/metadata/v2/testdata/container_stats.json

```diff
@@ -1,152 +1,154 @@
 {
-  "blkio_stats": {
-    "io_merged_recursive": [],
-    "io_queue_recursive": [],
-    "io_service_bytes_recursive": [
-      {
-        "major": 259,
-        "minor": 0,
-        "op": "Read",
-        "value": 12288
-      },
-      {
-        "major": 259,
-        "minor": 0,
-        "op": "Write",
-        "value": 144908288
-      },
-      {
-        "major": 259,
-        "minor": 0,
-        "op": "Sync",
-        "value": 8122368
-      },
-      {
-        "major": 259,
-        "minor": 0,
-        "op": "Async",
-        "value": 136798208
+  "470f831ceac0479b8c6614a7232e707fb24760c350b13ee589dd1d6424315d98": {
```

Comment:
can we test against two different fixtures? 1.4.0 and < 1.4.0

Reply:
these ones are going to be identical < 1.4.0 and 1.4.0 since it's not a container-specific one anymore.

Reply:
correct